code/modules/mob/living/simple_animal/guardian/types/toy.dm

```diff
@@ -0,0 +1,27 @@
+//Toy
+/mob/living/simple_animal/hostile/guardian/toy
+	melee_damage_lower = 0
+	melee_damage_upper = 0
+	obj_damage = 0
+	next_move_modifier = 0.1 //attacks 90% faster
+	playstyle_string = "<span class='holoparasite'>As a <b>toy</b> type you are absolutely useless in every way, and a total liability to your owner. but you look cool!</span>"
+	magic_fluff_string = "<span class='holoparasite'>..And draw the Clown, an utterly annoying and useless liability.</span>"
+	tech_fluff_string = "<span class='holoparasite'>Boot sequence complete. Standard combat modules locked. Holoparasite swarm online.</span>"
+	carp_fluff_string = "<span class='holoparasite'>CARP CARP CARP! You caught one! It's weak and useless. Can I have a refund?.</span>"
+	var/battlecry = "HONK"
+
+/mob/living/simple_animal/hostile/guardian/toy/verb/Battlecry()
+	set name = "Set Battlecry"
+	set category = "Guardian"
+	set desc = "Choose what you shout as you limply slap people."
+	var/input = stripped_input(src,"What do you want your battlecry to be? Max length of 6 characters.", ,"", 7)
+	if(input)
```

Comment:
You need to add a check for the chat filter.
```
if(CHAT_FILTER_CHECK(input))
    [some warning message]
    return
```

Reply:
sure.

Reply:
considering the pain the chat filter is putting me through here, i just wont include it so travis passes. i'll paste my shitcode and let you make your code suggestions. this bypass is already in the game so it'll need to get fixed either way, shrug. 

	if(CHAT_FILTER_CHECK(input))
    	to_chat(src, "<span class='holoparasite'>Your battlecry may not include forbidden words! Consider rereading the server rules.</span>")
    	return